client/client-multi/client-cpp/impl/utils/ThreadPool.cpp

```diff
@@ -114,32 +119,26 @@ void ThreadPool::shutdownNow()
 void ThreadPool::start()
 {
     for (std::size_t i = 0; i < workerCount_; ++i) {
-        workers_.push_back(std::thread(Worker(*this)));
+        workers_.emplace_back(Worker(*this));
```

Comment:
The `emplace_back()` is used to push multiple values.
You can use `push_back()` here, because since C++11 this method takes rvalue references as well:

    void push_back( T&& value );